app/controllers/users/passwords_controller.rb

```diff
@@ -7,13 +7,14 @@ def edit
     end
 
     def update
-      @update_user_password_form = UpdateUserPasswordForm.new(current_user)
-
-      result = @update_user_password_form.submit(user_params)
+      @update_user_password = UpdateUserPassword.new(
+        current_user, user_session, user_params[:password]
```

Comment:
I had them as named args when I started refactoring, but changed my mind because the args are already clearly named here, and it would have been repeating the argument names, such as `UpdateUserPassword.new(user: current_user, user_session: user_session, password: user_params[:password]`. I think named args are beneficial when you might call the class with ambiguous parameters, like strings and integers, or if the class is called in multiple places. Thoughts?